lib/response.js

```diff
@@ -1052,6 +1052,32 @@ function sendfile(res, file, options, callback) {
 }
 
 /**
+ * Run every tracers set at the application level for given events
+ * and arguments. To set a tracer on the application use the `instrument`
+ * method.
+ *
+ * @param {function} event The event to trace.
+ * @return {ServerResponse} for chaining
+ * @public
+ */
+
+res.trace = function(event) {
+  var args = [];
+  switch (arguments.length) {
+    case 0:
+      throw new Error('No event defined!');
+    case 1: break;
+    case 2:
+      args = [arguments[1]];
+      break;
+    default:
+      args = [].slice.call(arguments, 1);
+  }
+  this.app._callTracers(this, event, args);
```

Comment:
Re docs @frankrousseau I created a tracking issue a while back: https://github.com/expressjs/expressjs.com/issues/729.  If there questions or issues specific to how/what needs docs, please put them there.  Thanks!